mongo_connector/doc_managers/neo4j_doc_manager.py

```diff
@@ -33,13 +31,14 @@ class DocManager(DocManagerBase):
   def __init__(self, url, auto_commit_interval=DEFAULT_COMMIT_INTERVAL,
                  unique_key='_id', chunk_size=DEFAULT_MAX_BULK, **kwargs):
     self.remote_graph = Graph(url, **kwargs.get('clientOptions', {}))
-    graph = Graph()
+    self.graph = Graph()
     self.auto_commit_interval = auto_commit_interval
     self.unique_key = unique_key
     self.chunk_size = chunk_size
     if self.auto_commit_interval not in [None, 0]:
```

Comment:
Actually this was the default config. But makes sense changing the default. I'll check how I can make this; since it's managed by time, I think. Not by transanction number